CommonMark/CommonMark.py

```diff
@@ -1078,8 +1078,7 @@ def closeUnmatchedBlocks(self, already_done, oldtip):
         if blank and container.last_line_blank:
             self.breakOutOfLists(container, line_number)
 
-        while container.t != "ExtensionBlock" and \
```

Comment:
It causes 38 test failures on travis, see here: https://travis-ci.org/lu-zero/CommonMark-py/jobs/79385688

Probably due to our outdated spec.txt file.